base/src/main/java/edu/artic/base/utils/ActivityExtensions.kt

```diff
@@ -21,3 +19,23 @@ fun Activity.setWindowFlag(bits: Int, on: Boolean) {
     }
     win.attributes = winParams
 }
+
+fun Activity.isMyServiceRunning(serviceClass: Class<*>): Boolean {
+    val manager = getSystemService(Context.ACTIVITY_SERVICE) as ActivityManager
+    for (service in manager.getRunningServices(Integer.MAX_VALUE)) {
+        if (serviceClass.name == service.service.className) {
+            return true
+        }
+    }
+    return false
+}
+
+fun isServiceRunningInForeground(context: Context, serviceClass: Class<*>): Boolean {
+    val manager = context.getSystemService(Context.ACTIVITY_SERVICE) as ActivityManager
+    for (service in manager.getRunningServices(Integer.MAX_VALUE)) {
```

Comment:
for more kotlin idiomatic use, if you're using a `for loop` you're probably not doing it right. 
```
manager.getRunningServices(Int.MAX_VALUE).any { it.service.className == serviceClass.name }
```
Also don't use `Integer` (thats a java class) use Kotlin equivalent `Int`.